src/Components/Benefits.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import styled from 'styled-components'
+import Back from '../images/jade-stephens-N21356amsyw-unsplash.jpg'
+import Line from '../images/Line.svg'
+
+const BenefitsWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+  @media (min-width: 768px) {
+    flex-direction: row;
+    padding: 2rem;
+  }
+`
+
+const Text = styled.div`
+  background-image: url("../images/Blob.png");
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  color: green;
+  margin-top: 1rem;
```

Comment:
`Margin-top` property set to `1rem`, which is then overridden in the media query. It would be more efficient to only set this property once in the media query.